packages/core/src/interpreter.ts

```diff
@@ -118,6 +119,7 @@ export class Interpreter<
   // TODO: remove
   public _forwardTo: Set<AnyActorRef> = new Set();
 
+  public system: ActorSystem<any>;
```

Comment:
should the system be accessible publicly from just any ref belonging to it?

Reply:
According to early feedback, yes.